### ISSUE TITLE
Flexbox alignment & ordering classes shouldn't hardcode the grid tiers

### ISSUE THREAD

**Issue by marekkobida** [twbs/bootstrap#18026]
my-file.scss
```
	$enable-flex: true;

	$grid-breakpoints: (
	  xs: 0,
	  sm: 544px,
	  md: 768px,
	  lg: 992px
	);

	@import "../bootstrap/variables";

	@import "../bootstrap/mixins";

	@import "../bootstrap/normalize";
	@import "../bootstrap/reboot";

	@import "../bootstrap/grid";
```

If I remove `xl` from breakpoints variable, classes like

```
	.col-xl-top {
	    align-self: flex-start;
	}

	.col-xl-center {
	    align-self: center;
	}

	.col-xl-bottom {
	    align-self: flex-end;
	}
```

still remain in compiled code... Can you please fix that or tell me what I am doing wrong?

**Comment by cvrebert** [comment]
The code in https://github.com/twbs/bootstrap/blob/807fb8847aa8cbe54ad371fccdec93b2518c1558/scss/_grid.scss#L50-L122 needs to be refactored to use a loop over `$grid-breakpoints` instead of hardcoding the grid tier names.

**Comment by cvrebert** [comment]
Fixed by #18029.
